docs/git/terminal/lesson.md

```diff
@@ -27,7 +27,7 @@ By the end of this class, you should be able to:
 
 ## Before you start
 
-If you're on Windows, make sure that you've followed the instructions [here](../../js-core-1/preparation) to get setup with WSL. Alternatively, these exercises can be completed using [Git Bash](https://gitforwindows.org/)
+If you're on Windows, make sure that you've followed the instructions [here](https://syllabus.codeyourfuture.io/git/preparation) to get setup with WSL. Alternatively, these exercises can be completed using [Git Bash](https://gitforwindows.org/)
```

Comment:
Thanks @Salvacg737 !

This is pretty close but we'll want to make sure we're using a relative url to access the page

https://desktop.arcgis.com/en/arcmap/10.3/tools/supplement/pathnames-explained-absolute-relative-unc-and-url.htm

Also, we'll want to make sure it's pointing to [this page](https://syllabus.codeyourfuture.io/js-core-1/preparation#1-install-windows-subsystem-for-linux-60-minutes) of the Syllabus.

Have another try and give me a nudge when you've pushed your change! ☺